budgetportal/webflow/urls.py

```diff
@@ -21,5 +22,15 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^infrastructure-projects/provincial/(?P<id>\d+)-(?P<slug>[\w-]+)/csv-download$",
```

Comment:
now that it's django 2 thanks to you, you can do something like `<int:id>-<slug:slug>`